clients/python-wrapper/lakefs/config.py

```diff
@@ -0,0 +1,70 @@
+import os
+import yaml
+from pathlib import Path
+from typing import NamedTuple
+
+from lakefs_sdk import (
+    Configuration,
+)
+from lakefs.exceptions import NoAuthenticationFound
+
+_LAKECTL_YAML_PATH = os.path.join(Path.home(), ".lakectl.yaml")
```

Comment:
`Path / string` is functionally equivalent to  to `os.path.join`... whichever you prefer.

Reply:
Yes but using "/" as path delimiter will break in windows

Reply:
it will not.. 

```python
>>> print(pathlib.PureWindowsPath('a') / 'b')
a\b
```

Reply:
Must have been coding in go for too long 😅